cordapp/src/main/kotlin/net/corda/plugins/Signing.kt

```diff
@@ -1,23 +1,23 @@
 package net.corda.plugins
 
+import net.corda.plugins.SigningOptions.Companion.SYSTEM_PROPERTY_PREFIX
 import org.gradle.api.Action
 import org.gradle.api.model.ObjectFactory
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Nested
 import javax.inject.Inject
 
-/** JAR sign and control option. */
 open class Signing @Inject constructor(objectFactory: ObjectFactory) {
 
     @get:Input
-    var enabled: Boolean = true
+    var enabled: Boolean = System.getProperty(SYSTEM_PROPERTY_PREFIX + "enabled", "true").toBoolean()
 
```

Comment:
There's a little-known Java API called [Boolean.getBoolean(String)](https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#getBoolean(java.lang.String)) ... :wink:

Reply:
Indeed, however in this case I want to default to `true` if property is missing.